src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2683,16 +2683,16 @@ PeerImp::checkValidation(
     std::shared_ptr<STValidation> const& val,
     std::shared_ptr<protocol::TMValidation> const& packet)
 {
-    try
+    if (!cluster() && !val->isValid())
     {
-        // VFALCO Which functions throw?
-        if (!cluster() && !val->isValid())
-        {
-            JLOG(p_journal_.warn()) << "Validation is invalid";
-            charge(Resource::feeInvalidRequest);
-            return;
-        }
+        JLOG(p_journal_.debug()) << "Validation forwarded by peer is invalid";
+        charge(Resource::feeInvalidRequest);
+        return;
+    }
```

Comment:
See comment above: I'm not sure why this was moved outside the `try` block while the other was moved in.